src/js/components/Pagination/Pagination.js

```diff
@@ -54,18 +54,29 @@ const Pagination = forwardRef(
       totalPages,
     );
 
-    const middlePagesBegin = Math.max(
-      Math.min(
-        activePage - numMiddlePages,
-        totalPages - numEdgePages - numMiddlePages * 2 - 1,
-      ),
-      numEdgePages + 2,
-    );
-
+    if (numMiddlePages < 1) {
+      console.warn(`Property "numMiddlePages" should not be < 1.`);
```

Comment:
What happens if I do supply 0 or -value? Will it break? Or just render funny?

Reply:
I should check a `-` number. If you supply `0` it just renders funny:
```
< 1 ...   ... 10 >
```

Reply:
We could set a floor, where if they provide < 1, we set to 1

Reply:
That's the direction I was heading... Then the warning would read `Property "numMiddlePages" should not be < 1, setting to min of 1.` or something

Reply:
Great, let me have a go with that.